src/bootstrap3/utils.py

```diff
@@ -13,6 +14,10 @@
 
 from .text import text_value
 
+# TODO: Remove after support for Django 3.2 ends
+DJANGO3 = get_version() < "4"
```

Comment:
While it is correct for supported django versions, I think it might be clearer to use a name aligned to the check, i.e. something like 
```suggestion
IS_PRE_DJANGO_4 = get_version() < "4"
```

Reply:
Agreed.